internal/charm/utils.go

```diff
@@ -76,9 +76,12 @@ func SchemaFilesInCurrentDir(relativeDir string, fileExtensions []string) []stri
 		if !file.Type().IsDir() {
 			for _, ext := range fileExtensions {
 				if strings.HasSuffix(file.Name(), ext) {
+					if !strings.HasSuffix(relativeDir, "/") {
+						relativeDir += "/"
```

Comment:
Does this work on windows?

Reply:
Ah you might be right! I'll need `string(os.PathSeparator)`